### DISCUSSION TITLE
useCountDownHooks

### DISCUSSION THREAD

**Discussion by SceneryCN** [childrentime/reactuse#98] [Q&A] [answered]
这个Hooks里面使用了setInterval，跟aHooks一样，都没有使用requestAnimationFrame，- -表示很想了解为啥不用这个，不是性能更好而且不会出现时间偏差么- -

**Comment by childrentime** [comment, marked as answer]
1. 兼容性。 raf的兼容性没 setInterval好
2. 一般的倒计时不需要那么精确，不会频繁更新
3. raf在幕后不会执行

**Comment by childrentime** [comment]
还有就是，没有前端的倒计时可以不出现时间偏差，除非一直与后端的高精度时间同步，不然总会出现偏差。
